KesaApplication/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  KesaApplication
+//
+//  Created by Jiwoo Lim on 2016. 6. 1..
+//  Copyright © 2016년 kesa. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    //Function to display error message
+    func displayAlertMessage(userMessage: String){
```

Comment:
Can this function be **refactored** into **ErrorMessageHandler class**? It will be better to have same error message style through out the application.